sel4test-hw/build.py

```diff
@@ -66,8 +66,8 @@ def build_filter(build: Build) -> bool:
         # No MCS + SMP for platforms with global timer for now (see seL4/seL4#513)
         if plat.name == 'SABRE' and build.is_smp() and build.is_mcs():
             return False
-        # SCHED_CONTEXT_0014 fails on TX2 and ODROID_C4: https://github.com/seL4/seL4/issues/928
-        if (plat.name == 'TX2' or plat.name == 'ODROID_C4') and \
+        # SCHED_CONTEXT_0014 fails on TX1, TX2 and ODROID_C4: https://github.com/seL4/seL4/issues/928
+        if (plat.name == 'TX2' or or plat.name == 'TX1' or plat.name == 'ODROID_C4') and \
```

Comment:
```
        if (plat.name in ['TX', 'TX2', 'ODROID_C4']) and \
```

Reply:
Indeed, 3 is enough for a list. Will update.